vision/vae_mnist/vae_mnist.jl

```diff
@@ -6,10 +6,11 @@
 
 using Base.Iterators: partition
 using BSON
-using CUDAapi: has_cuda_gpu
+using CUDA
 using DrWatson: struct2dict
 using Flux
-using Flux: logitbinarycrossentropy, chunk
+using Flux: @functor, chunk
```

Comment:
looks like chunk is not used

Reply:
It is used in the convert_to_image function: https://github.com/FluxML/model-zoo/pull/267/files#diff-d2fcc7489d82937b5b466b70d629d200f23003b00a0e923cfa5031eedfa54d4cL70